File: node-fastify/entrypoint.js
const {
  PORT
} = process.env;

const server = require('fastify')();

server.get('/', (req, res) => {
  res.header('Content-Type', 'application/json');
  res.send('hello world');
});

server.listen(PORT || 3000, '0.0.0.0');

File: go-core/entrypoint.go
package main

import (
	"fmt"
	"log"
	"net/http"
)

func main() {
	http.HandleFunc("/", func(res http.ResponseWriter, req *http.Request) {
		res.Header().Set("Content-Type", "application/json")
		fmt.Fprint(res, "\"hello world\"")
	})
	log.Fatal(http.ListenAndServe(":3000", nil))
}
